CHANGES

```diff
@@ -1,5 +1,7 @@
 arpack-ng - 3.9.0
  
+* Avoid calling [c|z]dotc for better portability on macOS
```

Comment:
You forgot to add your name! (grouping changes by contributors)

Reply:
It didn't seem mandatory, see, e.g., https://github.com/opencollab/arpack-ng/blob/d5f793763f332f2dd0f43063dc38862534ad412d/CHANGES#L241.

Reply:
No but it helps to track/look-over changes! As you like

Reply:
I'm OK staying an anonymous John Doe 😄